lib/protocol/protocol.js

```diff
@@ -439,7 +439,7 @@ function driverShouldDoJwpProxy (driver, req, command) {
 
   // if it looks like we have an image element in the body, never proxy
   const stringBody = JSON.stringify(req.body);
```

Comment:
I'm still not quite sure about that. What if any other valid request includes accidentally includes '-image' substring in its body?

Reply:
it would have to include `appium-image-element-`. but you are right it is a risk. for a similar reason the webdriver spec does the whole `element-6066-11e4-a52e-4f735466cecf` thing.

one alternative is actually walking through the body recursively and seeing if we have an object with the exact form of an element. that seemed like it might be overkill. @imurchie what do you think?

Reply:
having such suffix might be useful IMHO